submissions/description/cross_site_scripting_xss/cross_site_request_forgery_csrf/flash_based/high_impact/template.md

```diff
@@ -0,0 +1,48 @@
+# Flash-Based Cross-Site Request Forgery (High Impact)
+
+## Overview of the Vulnerability
+
+Cross-Site Request Forgery (CSRF) occurs when requests to the application are submitted on behalf of an authenticated user without their knowledge via crafted, malicious code which can be in the form of a link the user clicks. The application is unable to distinguish between the malicious request and a legitimate request from the user.
+
+A high impact flash-based CSRF is possible for this application, allowing an attacker to submit requests to the application on behalf of an authenticated user. Additionally, the attacker needs to socially engineer the user to click on a link, or paste the malicious code into the user’s browser. If successful, the code will execute within that user’s browser in the context of this domain.
```

Comment:
This seems to be lacking what make sit High Impact vs Low Impact, as we need to specify what the difference is between them for the client and researcher.